src/test/miner_tests.cpp

```diff
@@ -181,9 +181,7 @@ void TestPackageSelection(const CChainParams& chainparams, CScript scriptPubKey,
 // NOTE: These tests rely on CreateNewBlock doing its own self-validation!
 BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
 {
-    // Disable size accounting (CPFP does not support it)
-    mapArgs["-blockmaxsize"] = strprintf("%u", MAX_BLOCK_SERIALIZED_SIZE);
-
+    // Note that by default, these tests run with size accounting enabled.
```

Comment:
Whoa, this global state was set without reverting it again at the end of the testcase? Please don't do that.

In any case ACK.